src/db/query.js

```diff
@@ -63,7 +64,10 @@ async function query(query, db = null, removeNull = false, source = 'modal') {
 
           for (let j = 0; j < e.columns.length; j++) {
             const column = e.columns[j]
-
+            let fieldData = res[j]
+            if (PLATFORM === "WINDOWS" && typeof fieldData == "string") {
+              fieldData = fieldData.split("''").join("'") // WTF!!!!
```

Comment:
It would make sense to do a `replace(/''/g, '"');` here for that extra performance